src/modules/hip/kernel/warp_perspective.hpp

```diff
@@ -0,0 +1,482 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+// -------------------- Set 0 - warp_perspective device helpers --------------------
+__device__ void warp_perspective_parcommon_hip_compute(float perspectiveMatrixElement, float4 locParCComponent_f4, d_float8 *locParC_f8)
+{
+    d_float8 increment_f8;
+    increment_f8.f4[0] = make_float4(0, perspectiveMatrixElement, perspectiveMatrixElement + perspectiveMatrixElement, perspectiveMatrixElement + perspectiveMatrixElement + perspectiveMatrixElement);
+    increment_f8.f4[1] = (float4)(perspectiveMatrixElement + increment_f8.f4[0].w) + increment_f8.f4[0];
+    locParC_f8->f4[0] = locParCComponent_f4 + increment_f8.f4[0];
```

Comment:
Please change the naming from locParC to something more intuitive as discussed
I actually not understanding the logic in this function
lets discuss on a call

Reply:
Updated